wicket-kendo-ui-culture/pom.xml

```diff
@@ -39,7 +39,12 @@
 			<version>${wicket.version}</version>
 			<scope>provided</scope>
 		</dependency>
-
+		<dependency>
+			<groupId>com.googlecode.wicket-jquery-ui</groupId>
+			<artifactId>wicket-kendo-ui</artifactId>
+			<version>${wicket.version}</version>
```

Comment:
Should be the current version: `6.20.1-SNAPSHOT`. Couls you also move it up a little, like the ordering in `wicket-kendo-ui/pom.xml`
This should be corrected for the build to pass, I guess...